reference/settings.v3.yml

```diff
@@ -1308,9 +1308,11 @@ paths:
       description: |-
         Updates SEO settings.
 
-         - Channel ID can be used as a query parameter for updating channel-specific settings. If omitted, you will interact with the global settings only. 
+         - You can use `channel_id` as a query parameter to update channel-specific settings. If omitted, you will only interact with the global settings. 
+         
+         - You can use `channel_id` in the request body of a headless channel of this endpoint to control the `www_redirect` settings for the headless channel's unique checkout URL.
 
-         - `null` should be supplied to delete overrides per given channel and to inherit values from global level. Partial updates are not supported and all settings should be supplied with `null` value in order to delete overrides per channel.
+         - `null` should be supplied to delete overrides per given channel and to inherit values from a global level. Partial updates are not supported, and all settings should be supplied with a `null` value to delete overrides per channel.
```

Comment:
@bc-traciporter btw partial updates are supported on this endpoint. So we can remove this piece 

> Partial updates are not supported, and all settings should be supplied with a `null` value to delete overrides per channel. 

and only keep 

> `null` should be supplied to delete overrides per given channel and to inherit values from a global level.